aces/pipeline_scripts/override_tclean_commands.json

```diff
@@ -1822,18 +1852,20 @@
         "uvtaper": [
           ""
         ],
-        "niter": 30000000,
-        "gain": 0.1,
+        "usemask": "pb",
+        "pbmask": 0.5,
+        "niter": 50000000,
+        "cyclefactor": 3,
+        "nchan": 3836,
+        "width": "4882495Hz",
```

Comment:
Are you sure this is right?  4.8 MHz?  The previous value was 0.48 MHz

Reply:
thanks. should have been "488249.5Hz".   will fix in new PR

Reply:
OK, generally it's better to correct the open PR rather than open a fresh one, though.